src/platform/capslock_osx.cpp

```diff
@@ -0,0 +1,29 @@
+/*
+    Copyright © 2016 by The qTox Project
+
+    This file is part of qTox, a Qt-based graphical interface for Tox.
+
+    qTox is libre software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    qTox is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with qTox.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include <QtCore/qsystemdetection.h>
+#if defined(__APPLE__) && defined(__MACH__)
+#include "src/platform/capslock.h"
+
+bool Platform::capsLockEnabled()
+{
+    return false;
```

Comment:
um, I would think that osx does support caps-lock detection – could you at least add `TODO: implement for osx` ?